execution.bs

```diff
@@ -3379,37 +3379,35 @@ namespace std {
 in_place_stop_token() noexcept;
 </pre>
 
-1. *Effects*: initializes `source_` with `nullptr`.
-
-<pre highlight=c++>
-explicit in_place_stop_token(const in_place_stop_source* source) noexcept;
-</pre>
-
-1. *Effects*: initializes `source_` with `source`.
+1. *Effects*: initializes <code><i>source_</i></code> with `nullptr`.
 
 <pre highlight=c++>
 void swap(stop_token& rhs) noexcept;
 </pre>
 
-2. *Effects*: Exchanges the values of `source_` and `rhs.source_`.
+2. *Effects*: Exchanges the values of <code><i>source_</i></code> and <code>rhs.<i>source_</i></code>.
 
 #### Members <b>[stoptoken.inplace.mem]</b> #### {#spec-stoptoken.inplace.mem}
 
 <pre highlight=c++>
 [[nodiscard]] bool stop_requested() const noexcept;
 </pre>
 
-1. Returns: `source_ != nullptr && source_->stop_requested()`.
+1. *Effects*: Equivalent to: <code>return <i>source_</i> != nullptr && <i>source_</i>->stop_requested();</code>
 
-2. *Remarks*: If `source_ != nullptr`, then any calls to this function must strongly happen before the beginning of invocation of the destructor of `*source_`.
+2. [*Note*: The behavior of `stop_requested()` is undefined unless the call
+    strongly happens before the start of the destructor of the associated
+    `in_place_stop_source`, if any ([basic.life]). --*end note*]
 
 <pre highlight=c++>
 [[nodiscard]] bool stop_possible() const noexcept;
 </pre>
 
-3. *Returns*: `source_ != nullptr && source_->stop_possible()`.
+3. *Effects*: Equivalent to: <code>return <i>source_</i> != nullptr;</code>
 
-2. *Remarks*: If `source_ != nullptr`, then any calls to this function must strongly happen before the beginning of invocation of the destructor of `*source_`.
+4. [*Note*: The behavior of `stop_possible()` is implementation-defined unless
+    the call strongly happens before the end of the storage duration of the
+    associated `in_place_stop_source` object, if any ([basic.stc.general]). --*end note*]
```

Comment:
Yes, I think that fits with the core wording. Is this useful enough semantics for the user, or do we need to play tricks with std::uintptr_t to avoid the implementation-defined behavior resulting from the indeterminate pointer value?

Reply:
And thanks @jensmaurer :-)